resources/lrsql/config/prod/default/webserver.edn

```diff
@@ -6,6 +6,11 @@
  :key-enable-selfie         #boolean #or [#env LRSQL_KEY_ENABLE_SELFIE true]
  :jwt-exp-time              #long #or [#env LRSQL_JWT_EXP_TIME 3600]
  :jwt-exp-leeway            #long #or [#env LRSQL_JWT_EXP_LEEWAY 1]
+ :jwt-no-val                #boolean #or [#env LRSQL_JWT_NO_VAL false]
+ :jwt-no-val-uname          #or [#env LRSQL_JWT_NO_VAL_UNAME nil]
+ :jwt-no-val-issuer         #or [#env LRSQL_JWT_NO_VAL_ISSUER nil]
+ :jwt-no-val-role-key       #or [#env LRSQL_JWT_NO_VAL_ROLE_KEY nil]
+ :jwt-no-val-role           #or [#env LRSQL_JWT_NO_VAL_ROLE nil]
```

Comment:
Perhaps these `:jwt-no-val-*` config vars should have defaults after all, for example `:jwt-no-val-issuer` should be `"iss"` by default? And those default values would have no effect if `:jwt-no-val` is `false`, so they won't affect normal usage.

Reply:
no, none of the defaults are universal at all and half the point of my approach was to keep DOD bullshit out of codebase.